src/plugins/ui_actions/public/actions/action.ts

```diff
@@ -92,7 +92,7 @@ export interface Action<Context extends BaseContext = {}, T = ActionType>
    * Returns a title to be displayed to the user.
```

Comment:
If it helps, I can change this comment to inform users that a React element is also supported here.

Reply:
Yeah, that'd be helpful.